### ISSUE TITLE
unable to connect to v .23 frontend?

### ISSUE THREAD

**Issue by GoogleCodeExporter** [pot8oe/mythmote#34]
```
-configured mythtv frontend to allow network control (default port)
-verified that port was not being blocked by frontend/backend machine
-verified Android phone (G1) could ping IP address of frontend
- tried connecting from G1 to frontend -- "i/o error connecting to host..."

Does 1.2.1 of mythmote just not work with a mythtv v.23 backend yet?  or am I 
missing something else obvious?

-frontend (and backend) are running Mythbuntu
-mythmote is running on a G1 (latest official update from Tmobile)

The G1 is connected to my wireless network and can definitely ping the frontend 
machine with no problem.


```

Original issue reported on code.google.com by `thirsty...@gmail.com` on 23 Jun 2010 at 3:00
- Merged into: #16
